_includes/scripts/tikzjax.liquid

```diff
@@ -1,3 +1,8 @@
 {% if page.tikzjax %}
-  <script defer src="https://tikzjax.com/v1/tikzjax.js"></script>
+  <script
+    defer
+    src="https://tikzjax.com/v1/tikzjax.js"
```

Comment:
I'm wondering if this would not break offline mode?
The `src` seems to be hardcoded.

Reply:
There's nothing much we can do about tikzjax. Converting it to be 100% offline would involve too much change to its original source code.